bullet_train/lib/bullet_train/resolver.rb

```diff
@@ -61,9 +60,8 @@ def run(eject: false, open: false, force: false, interactive: false)
         end
 
         if interactive && !open
-          puts "\nWould you like to open `#{source_file[:absolute_path]}`? (y/n)\n"
+          print "\nWould you like to open `#{source_file[:absolute_path]}`? (y/n): "
           input = $stdin.gets
-          $stdin.getc while $stdin.ready?
```

Comment:
Can I ask your rationale for removing this line? Not against the change, just curious as to why.

Reply:
Great question! The mocked `StringIO` doesn't have `ready?` method so it was failing.

From [this SO post](https://stackoverflow.com/questions/15690902/how-to-clean-stdin-in-ruby), `$stdin.getc while $stdin.ready?` can be used to make sure all characters passed to STDIN are being read.

So...I should probably add it back here where user is copy-pasting:
https://github.com/bullet-train-co/bullet_train-core/pull/350/files/0c1a30879f9ff9d82006b9af46ba10899f0900d3#diff-169057f5b52e3c8d1a1f9510a4ee51d00c3974ae4b89567cd707bede5b73322fL42

I'll see how I can mock `$stdin.ready?`.

Reply:
Great, I wasn't sure about this portion myself so I'm glad you found the post.